srcalc/src/main/java/gov/va/med/srcalc/domain/model/ValueMatcher.java

```diff
@@ -96,9 +101,62 @@ public Expression getParsedExpression()
         return fBooleanExpression;
     }
     
+    /**
+     * Returns true if this matcher should be evaluated and false if it should not
+     * be evaluated.
+     */
+    @Basic
+    public boolean isEnabled()
+    {
+        return fEnabled;
+    }
+    
+    void setEnabled(final boolean enabled)
```

Comment:
It might make more sense to make this (and the expression) a true mutable property.